test/health_cards/issuer_test.rb

```diff
@@ -7,26 +7,35 @@
 class IssuerTest < ActiveSupport::TestCase
   setup do
     @key_path = Rails.application.config.well_known.jwk[:key_path]
+    @file_store = HealthCards::FileKeyStore.new(@key_path)
+    @issuer = HealthCards::Issuer.new(@file_store)
   end
 
   teardown do
-    FileUtils.rm_rf @key_path
+    FileUtils.rm_rf File.join(@key_path, HealthCards::FileKeyStore::FILE_NAME)
   end
 
-  test 'Creates keys' do
-    issuer = HealthCards::Issuer.new @key_path
-    jwks = issuer.jwks
+  test 'creates keys' do
+    jwks = @issuer.jwks
 
-    assert_path_exists(issuer.signing_key_path)
+    assert_path_exists(@file_store.key_path)
     assert_equal 1, jwks[:keys].length
-    assert(jwks[:keys].one? { |key| key[:use] == 'sig' && key[:alg] == 'ES256' })
+    jwks[:keys].one? do |key|
+      assert_equal 'sig', key['use']
+      assert_equal 'ES256', key['alg']
+    end
+  end
+
+  test 'created signed jwt' do
+    vc = HealthCards::VerifiableCredential.new({})
+    signed_jwt = @issuer.sign(vc, 'http://example.com')
+    assert_instance_of String, signed_jwt
```

Comment:
Not sure if right test to put it in but can probably check that the `signed_jwt` verifies with the public key and the credential is the same.
```
    assert_equal vc.credential.as_json, JSON::JWT.decode(signed_jwt, @issuer.public_key)
```

Reply:
Good call, will add